ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -136,7 +136,6 @@ Verify User Can Create A PV Storage
     ...                MAIN GOAL OF THE TEST CASE IS COVERED BY ODS-1814
     ${pv_name}=    Set Variable    ${PV_BASENAME}-A
     ${ns_name}=    Get Openshift Namespace From Data Science Project   project_title=${PRJ_TITLE}
-    Open Data Science Project Details Page       project_title=${PRJ_TITLE}    tab_id=workbenches
```

Comment:
Can you please explain the workflow so I understand why these lines aren't needed anymore in all of these places around? Is it because the Dasboard now opens the DS projects overview by default and we'll create a workbench right from that page?

Reply:
`Create Workbench` already calls `Open Data Science Project Details Page   project_title=${PRJ_TITLE}    tab_id=workbenches`